gwh.py

```diff
@@ -42,14 +42,14 @@ def index():
                 'homepage': payload['repository']['homepage'],
             }
             repo = repos.get(repo_meta['homepage'], None)
-            private_token = repo.get('private_token', None)
+            if not repo:
+                return "Nothing to do for " + repo_meta['homepage']
```

Comment:
[``](url)
```suggestion
                return "Nothing to do for " + repo_meta['homepage'], 422
```

Reply:
I guess return values are all over the map, actually, so never mind.

Reply:
I wonder whether 200 is just fine here, but that's matter fo taste. I guess both 200 or 422 are fine dending on the PoV.